cli/src/stake.rs

```diff
@@ -2006,13 +2006,6 @@ pub fn process_split_stake(
         let minimum_balance =
             rpc_client.get_minimum_balance_for_rent_exemption(StakeStateV2::size_of())?;
 
-        if lamports < minimum_balance {
-            return Err(CliError::BadParameter(format!(
-                "need at least {minimum_balance} lamports for stake account to be rent exempt, \
-                 provided lamports: {lamports}"
-            ))
-            .into());
-        }
```

Comment:
This was incorrect as of https://github.com/solana-labs/solana/pull/33295
It could be updated to check minimum stake amount, instead of removing. Thoughts?

Reply:
wouldn't hurt

Reply:
👍 will push shortly.

Reply:
I added it to the beginning of the `if !sign_only` block, to distinguish it from the RER check.